packages/patrol/lib/src/native/native_automator.dart

```diff
@@ -42,12 +44,17 @@ enum Binding {
 class NativeAutomator {
   /// Creates a new [NativeAutomator].
   NativeAutomator({
-    this.timeout = const Duration(seconds: 10),
+    this.connectionTimeout = const Duration(seconds: 20),
+    this.findTimeout = const Duration(seconds: 10),
     _LoggerCallback logger = _defaultPrintLogger,
     String? packageName,
     String? bundleId,
     Binding binding = Binding.patrol,
-  })  : _logger = logger,
+  })  : assert(
+          connectionTimeout > findTimeout,
+          'find timeout is longer than connection timeout',
```

Comment:
Corner case: should this throw when `connectionTimeout == findTimeout`?

Reply:
Thanks – I didn't give it any thought.

The answer is yes.